docs/openshift-version-in-the-report.md

```diff
@@ -0,0 +1,73 @@
+# OpenShift Version in the Report
+
+The chart-verifier tool adds the OpenShift version where chart-testing has run
+in the report.  The information is available as a metadata at this path:
```

Comment:
Remove the extra blank space before `The information`

Reply:
I used to use two spaces.
Here is some of the reasons: https://stevelosh.com/blog/2012/10/why-i-two-space/ (Not my blog post)

Reply:
What is the benefit of a single space?

Reply:
RH style guide and writing guidelines says so. Also, the majority of style guides now prescribe the use of a single space after terminal punctuation in final written works and publications. We should update our writing to reflect current tools and an extra space between sentences creates a river of white in documents, diminishing the impact.

Please read: [https://www.cultofpedagogy.com/two-spaces-after-period/](https://www.cultofpedagogy.com/two-spaces-after-period/) and [https://www.instructionalsolutions.com/blog/one-space-vs-two-after-period](https://www.instructionalsolutions.com/blog/one-space-vs-two-after-period). Thanks!

Reply:
Is there a tool to format a file according to the RH Sytle guide?